extensions/interactions/FractionInput/FractionInput.py

```diff
@@ -78,3 +80,400 @@ class FractionInput(base.BaseInteraction):
         'calculation_id': 'Top10AnswerFrequencies',
         'addressed_info_is_supported': True,
     }]
+
+    @classmethod
+    def to_proto(cls, interaction):
+        """Creates a FractionInputInstance proto object.
+
+        Args:
+            interaction: InteractionInstance. The interaction instance
+                associated with this state.
+
+        Returns:
+            FractionInputInstance. The FractionInputInstance proto object.
+        """
+        customization_args_proto = (
+            cls._to_customization_args_proto(
+                interaction.customization_args))
+
+        outcome_proto = interaction.default_outcome.to_proto()
+
+        solution_proto = cls._to_solution_proto(
+            interaction.solution)
```

Comment:
```suggestion
        solution_proto = cls._to_solution_proto(interaction.solution)
```

in a lot of other places the code is put on multiple lines even when it would be possible to place it at one line, can you please fix that?

Reply:
Done